src/Screens/ConsultantProfile.tsx

```diff
@@ -221,9 +220,9 @@ const ConsultantProfile = ({navigation}: any) => {
           <View style={styles.picture}>
             <Image
               source={{
-                uri: consultantProfile?.profilePicture
-                  ? consultantProfile?.profilePicture
-                  : 'https://cdn.pixabay.com/photo/2015/10/05/22/37/blank-profile-picture-973460_1280.png',
+                uri: consultantProfile.profilePicture
+                  ? consultantProfile.profilePicture
+                  : defaultProfilePicture,

```

Comment:
use this:
uri:  iconsultantProfile?.profilePicture | | defaultProfilePicture
instead of this:
uri: consultantProfile?.profilePicture ? consultantProfile?.profilePicture : defaultProfilePicture,